tests/ReflectionTypeTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+namespace Go\ParserReflection;
+
+class ReflectionTypeTest extends \PHPUnit_Framework_TestCase
+{
+    protected function setUp()
+    {
+        if (PHP_VERSION_ID >= 70000) {
+            include_once (__DIR__ . '/Stub/FileWithClasses70.php');
+        }
+        if (PHP_VERSION_ID >= 70100) {
+            include_once (__DIR__ . '/Stub/FileWithClasses71.php');
+        }
+    }
+
+    /**
+     * Testing convertToDisplayType() with native \ReflectionType
+     *
+     * We're already testing it with Go\ParserReflection\ReflectionType
+     * elsewhere.
+     */
+    public function testTypeConvertToDisplayTypeWithNativeType()
+    {
+        if (PHP_VERSION_ID < 70000) {
+            $this->markTestIncomplete('PHP >= 7.0 implements ReflectionType');
```

Comment:
If PHP 7.0+ implements `ReflectionType`, then test must be marked as skipped on other PHP versions, not an incomplete one.